packages/mermaid/src/diagrams/sequence/svgDraw.js

```diff
@@ -703,7 +703,7 @@ export const insertArrowHead = function (elem) {
     .append('defs')
     .append('marker')
     .attr('id', 'arrowhead')
-    .attr('refX', 9)
+    .attr('refX', 7.9)
```

Comment:
`7.9` seems like a very weird number. I'm guessing this is the number that looks the best, but I just wanted to double-check that you didn't accidentally make a typo.

Reply:
Nope, arrived at that value after the extremely scientific method of zooming in and looking closely while adjusting the value. 🧑‍🔬